modules/arm_plugin/README.md

```diff
@@ -25,18 +25,35 @@ git clone --recurse-submodules --single-branch --branch=master https://github.co
  ```
 cd openvino_contrib/modules/arm_plugin
 ```
-3. Build the plugin in Docker* container:
+3. Build a Docker* image:
 ```
 docker image build -t arm-plugin -f Dockerfile.RPi32 .
 ```
-> **NOTE**: Docker* uses cache when building an image. `arm-plugin` image clones `opencv`, `openvino` and `openvino_contrib` repositories. 
-If you build `arm-plugin` image next time, Docker* does not clone repositories, existing image from the cache will be used instead. 
-If you want to clone repositories again to pull the latest changes, you need to add `--no-cache` option on the `docker image build` command.
-4. Export the archive with artifacts to the current directory:
+4. Build the plugin in Docker* container:
+
+Build process is performed by `/armplg_build.sh` script executed inside `/armcpu_plugin` directory(default container command to execute).
+All intermediate results and build artifacts are stored inside working directory.
+
+So one could either mount the whole working directory to get all results stored outside of the container
+```
+mkdir build
+docker container run --rm -ti -v $PWD/build:/armcpu_plugin arm-plugin
+```
+or export only the archive with artifacts
 ```
-docker run --rm -ti -v $PWD:/remote arm-plugin cp ./OV_ARM_package.tar.gz /remote
+docker container run --rm -ti --tmpfs /armcpu_plugin:rw -v $PWD:/remote arm-plugin sh -c "/armplg_build.sh && cp ./OV_ARM_package.tar.gz /remote"
 ```
-5. Extract the archive to `build` directory
+> **NOTE**: There are a few environment variables that control `/armplg_build.sh` script execution.
+>
+> *BUILD_JOBS* control number of threads for simultaneous compilation
+>
+> *BUILD_TYPE* control Debug/Release configuration (Release by default)
+>
+> *UPDATE_SOURCES* control sources retrievement
+>  - *clean*  - don't reload sources if already loaded, just clean build folders(default)
+>  - *reload* - delete all loaded sources and retrieve them again
+>  - *check*  - don't reload or cleanup sources
+6. Extract the archive to `build` directory
```

Comment:
Probably it would be helpful to add "on target ARM platform" to the end? 
Just to highlight the fact that it's cross-compilation and no need to run binaries on build machine.

Reply:
It's the reasonable point, however it doesn't make sense to call destination directory `build` for target platform.
Also shell command for deployment to target platform probably would be different and platform dependent.
So IMO this step should be just: "6. Extract the archive to target ARM platform"

Reply:
Agree. We may delete `mkdir` command and add `--one-top-level` to tar command, so all content is extracted into `OV_ARM_package`.

Reply:
Fixed